### ISSUE TITLE
Glycan identification in "combined_modified_peptide.tsv" file

### ISSUE THREAD

**Issue by mwfoster** [Nesvilab/FragPipe#1224]
Hi,

We are performing glycopeptide identification and quantification across many samples. The data in the "assigned modifications" column of the "combined_modified_peptide.tsv" file gives only the mass of the glycan, not the ID. I gather from a previous post that the "psm.tsv" contains the information to connect the "assigned modification" to the "observed modification" that would be found in the glycan database. However, since the psm.tsv is specific to a sample, this could require performing this lookup on a large number of psm.tsv files. Is there an easier way to do this, or could you modify the "combined_modified_peptide.tsv" to include the best match to the glycan database?

Thanks,

Matt

**Comment by dpolasky** [comment]
Hi Matt,

There is not an easier way to do this at the moment, unfortunately. I think we could add the Observed Modifications column to the combined_modified_peptide.tsv @fcyu? Or we may need to discuss internally the best way to propagate the glycan information. 

Best,
Dan

**Comment by fcyu** [comment]
Yes, we can let IonQuant take that information and propagate to the combined table.

Best,

Fengchao

**Comment by mwfoster** [comment]
Thanks. That would be very helpful. Regards, Matt

**Comment by ShinjaChakraborty** [comment]
Hi, could you please provide me the detailed protocol to run Fragpipe for N glycopeptides identification?

**Comment by fcyu** [comment]
Hi @ShinjaChakraborty , you could start with one of the built-in glyco-N workflows. Here is a tutorial if you need it: https://fragpipe.nesvilab.org/docs/tutorial_glyco.html

Best,

Fengchao

**Comment by ShinjaChakraborty** [comment]
I am getting error in Proteinprophet
[log_2024-08-09_23-27-33.txt](https://github.com/user-attachments/files/16568314/log_2024-08-09_23-27-33.txt)

**Comment by fcyu** [comment]
Same reason, there are no IDs
```shell
WARNING: Mixture model quality test failed for charge (2+).
WARNING: Mixture model quality test failed for charge (3+).
WARNING: Mixture model quality test failed for charge (4+).
WARNING: Mixture model quality test failed for charge (5+).
WARNING: Mixture model quality test failed for charge (6+).
WARNING: Mixture model quality test failed for charge (7+).

...read in 0 1+, 0 2+, 0 3+, 0 4+, 0 5+, 0 6+, 0 7+ spectra with min prob 0.05
```

Best,

Fengchao
